libs/server/Resp/RespServerSession.cs

```diff
@@ -185,7 +185,7 @@ public RespServerSession(
             bool enableScripts)
             : base(networkSender)
         {
-            this.customCommandManagerSession = new CustomCommandManagerSession(storeWrapper.customCommandManager);
+            this.customCommandManagerSession = storeWrapper.serverOptions.LoadModuleCS != null ? new CustomCommandManagerSession(storeWrapper.customCommandManager) : null;
```

Comment:
Good catch! However, you have to update the unit tests also to enable LoadModuleCS because the ones using modules are breaking.

Reply:
I dont think we want to start using Lazy in the codebase just for this. we have fixed the underlying cause of the large overhead, so the array should no longer be a bottleneck.